### PR TITLE
add 1.14.6-kustomize-3.4.0 image

### PR DESCRIPTION
**Pull request by shunyy**
added image with

- kubectl: 1.14.6
  - https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
  - snapshot: 

- kustomize: 3.4.0
  - https://github.com/kubernetes-sigs/kustomize/releases

### CODE REVIEW

**Review by mgi166 — APPROVED** [review]
LGTM

File: 1.14.6-kustomize-3.4.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ && openssl sha1 -sha256 kubectl \
+ && chmod +x ./kubectl \
+ && mv ./kubectl /usr/local/bin/kubectl \
+ && rm -rf kubectl.sha256 \
+ && : install aws-iam-authenticator \
+ && curl -O https://amazon-eks.s3-us-west-2.amazonaws.com/$K8SVERSION/$RELEASEVER/bin/linux/amd64/aws-iam-authenticator \
+ && curl -O https://amazon-eks.s3-us-west-2.amazonaws.com/$K8SVERSION/$RELEASEVER/bin/linux/amd64/aws-iam-authenticator.sha256 \
+ && openssl sha1 -sha256 aws-iam-authenticator \
+ && chmod +x ./aws-iam-authenticator \
+ && mv ./aws-iam-authenticator /usr/local/bin/aws-iam-authenticator \
```
Comment (civitaspo) [inline 1.14.6-kustomize-3.4.0/Dockerfile]: This is not required anymore, I think.

Comment (shunyy) [reply 1.14.6-kustomize-3.4.0/Dockerfile]: :ha-n:
https://dev.classmethod.jp/cloud/aws/eks-update-get-token-cmd/

Thanks!

Comment (shunyy) [reply 1.14.6-kustomize-3.4.0/Dockerfile]: fixed 😸

**Review by civitaspo — APPROVED** [review]
Ship it!

**Pull request merged**